modules/media/ajax/FileDownload.php

```diff
@@ -15,6 +15,8 @@
  */
 
 $user =& User::singleton();
```

Comment:
Should be `\User` to stay consistent

Reply:
It's out of scope for this PR but I'll create an issue to address this generally.

Reply:
@paiva #4030